ghedesigner/radial_numerical_borehole.py

```diff
@@ -24,26 +27,8 @@ class RadialNumericalBH(object):
     Energy Storage-EcoStock. Pomona, NJ, May 31-June 2.
     """
 
-    def __init__(
-            self,
-            single_u_tube: SingleUTube,
-            ground_init_temp: float = 20.0,
-            data_type: np.dtype = np.double,
-    ):
+    def __init__(self, single_u_tube: SingleUTube):
         self.single_u_tube = single_u_tube
-        self.dtype = data_type
-
-        self.cell_inputs = {
-            "type": 0,
-            "inner-radius": 1,
-            "center-radius": 2,
-            "outer-radius": 3,
-            "thickness": 4,
-            "conductivity": 5,
-            "heat-capacity": 6,
-            "initial-temperature": 7,
-            "volume": 8,
-        }
```

Comment:
Some of these were unused, and rather than storing them on a `dict`, an `enum` is a better object for this functionality. I created an enum class above to replace this.